test/Atmos/EDMF/edmf_kernels.jl

```diff
@@ -1259,10 +1259,25 @@ function turbconv_normal_boundary_flux_second_order!(
     bc::EDMFBottomBC,
     atmos::AtmosModel,
     fluxᵀn::Vars,
-    _...,
+    args,
 )
-    nothing
+    @unpack state⁻, aux⁻, diffusive⁻, hyperdiff⁻, t, n⁻ = args
+    en_flx = fluxᵀn.turbconv.environment
+    tend_type = Flux{SecondOrder}()
+    _args⁻ = (;
+        state = state⁻,
+        aux = aux⁻,
+        t,
+        diffusive = diffusive⁻,
+        hyperdiffusive = hyperdiff⁻,
+    )
+    pargs = merge(_args⁻, (precomputed = precompute(atmos, _args⁻, tend_type),))
+    total_flux =
+        Σfluxes(en_ρatke(), eq_tends(en_ρatke(), atmos, tend_type), atmos, pargs)
+    nd_ρatke = dot(n⁻, total_flux)
+    en_flx.ρatke += nd_ρatke
```

Comment:
If this condition prescribes the flux at plus from the flux at minus, should this just be  `en_flx.ρatke = nd_ρatke`? Also, it  might be good to comment this intended behavior above `function turbconv_normal_boundary_flux_second_order!`